_component_design/table-expansion.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Table Expansion
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table rows can be expanded to show the user additional data. By hiding some table data that can only be accessed at the request of the user, expanded tables provide for more data rich layouts without increasing the complexity for the user.
+preview-image: preview-images/tables-nested.svg
+status: stable
+resource: true
+last-modified: 2018-12-03
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use expanded tables when you want to include additional levels of data within a table.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct expanded tables:
+
+- All rows should be collapsed by default.
+- The expanded area inherits the styling of the parent row (for example, bound or unbound and zebra-stripe color).
+- Rows remain expanded until collapsed by the user.
+- There can only be one expandable area per row.
+- A user can expand a row by:
+  - Clicking anywhere in the row that is not a link.
+  - Clicking the angle-down icon.
+  - Clicking a badge icon.
+- A user can collapse a row by:
+  - Clicking anywhere in the parent row.
+  - Clicking directly on the angle-up icon in a parent row or at the bottom of an expanded area.
+- In addition to the parent row, include an angle-up icon in the bottom right corner of the expanded area.
+- The viewport remains unchanged with expansion and collapse.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Explanatory text specifications
+
+Use this variation of an expandable table to add additional supporting text to a table row.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A table that includes explanatory text contains the following elements:
+
+- **Expand (required)**: The table row expands when the user clicks the angle-down icon.
+- **Explanatory text (required)**: Explanatory text appears when the table row expands. Text can be a single line or can wrap to multiple lines.
```

Comment:
Make more generic and call out 'content area' - and then provide some info about text or kvp. Keep aggregation separate.

Reply:
done.